modules/registry/registry_test.go

```diff
@@ -9,6 +9,7 @@ import (
 
 	"github.com/cpuguy83/dockercfg"
 	"github.com/stretchr/testify/require"
+	"golang.org/x/crypto/bcrypt"
```

Comment:
bug: could you run `go mod tidy` from the root of the registry module? The lint phase detected the entry is missing in the go.mod file

Reply:
Whatever works for you sounds good to me 😊